content/en/blog/_posts/2023-12-18-node-expand-secret-moves-ga

```diff
@@ -0,0 +1,94 @@
+---
+layout: blog
+title: "Kubernetes 1.29: CSI Storage Resizing Authenticated and Ready for GA in v1.29"
+date: 2023-12-18
+slug: csi-node-expand-secret-support-ga
+---
+**Authors:** Humble Chirammal, Louis Koo (deeproute.ai)
+
+In the dynamic universe of Kubernetes, version v1.29 unfolds a momentous chapter
+as support for authentication during CSI (Container Storage Interface) storage
+resize operations ascends from Beta to General Availability (GA).
```

Comment:
```suggestion
Kubernetes version v1.29 brings generally
available support for authentication during CSI (Container Storage Interface) storage
resize operations.
```

Usually I'm OK with whatever intro folks want, but this didn't ring true. For a lot of clusters this is a niche feature (I personally haven't worked with anyone who used the beta feature); I know it'll be popular with its target audience but “momentous” feels several notches beyond too strong.
So: please reword. I added a suggestion.